src/MarketingPageAcceptanceTests/MarketingPageAcceptanceTests.Actions/Pages/Dashboard.cs

```diff
@@ -53,7 +53,7 @@ public void NavigateToSection(string sectionTitle)
         /// </summary>
         public void NavigateToPreviewPage()
         {
-            driver.FindElement(pages.Dashboard.PreviewPage).Click();
+            driver.FindElement(pages.Dashboard.PreviewPage).FindElement(By.ClassName("nhsuk-button")).Click();
```

Comment:
As the dashboard only has a button within the `pages.Dashboard.PreviewPage` section, this could probably be condensed down